scene-lib/src/scenelib/annotations/el/ATypeElementWithType.java

```diff
@@ -80,7 +80,7 @@ public boolean equals(Object o) {
             && ((ATypeElementWithType) o).equals(this);
     }
 
-    public boolean equalsTypeElementWithType(ATypeElementWithType o) {
+    public boolean equals(ATypeElementWithType o) {
```

Comment:
I think it would be nicer to remove this method and merge the logic into the existing `equals` method.
Having overloads of `equals` is error-prone and as nobody uses the overloaded method it would be safe to remove it.